.eslintrc.js

```diff
@@ -66,20 +66,20 @@ module.exports = {
       },
     ],
     // rely on TypeScript types to be the source of truth, minimize verbosity in comments
-    'jsdoc/require-param-type': ['off'],
     'jsdoc/require-param-description': ['error'],
+    'jsdoc/require-param-type': ['off'],
```

Comment:
FYI - I think this comment https://github.com/ionic-team/stencil/pull/4359/files/45309a3da216b947d8954daf072ab24466bc736e..a87503149155bc39316303d29c574920cc8f569d#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R68 got detached from this rule

Reply:
I think I pushed an intermediate commit that's not there anymore, afaict it's good now

Reply:
Hmm...I don't think I'm seeing it:

![Screenshot 2023-05-05 at 10 22 43 AM](https://user-images.githubusercontent.com/1930213/236485295-fe8f8e3c-43b4-44b3-a540-6161a0b08490.png)